src/kmod/cb-banning.c

```diff
@@ -319,6 +319,18 @@ void cbSetIgnoredProcess(pid_t pid)
 	}
 }
 
+void cbClearIgnoredProcess(pid_t pid)
+{
+	int64_t max = atomic64_read((atomic64_t *)&g_cb_ignored_pid_count);
+
+	if (max) {
+		atomic64_set((atomic64_t *)&g_cb_ignored_pid_count, 0);
+		memset(&g_cb_ignored_pids[0], 0,
```

Comment:
I just have a couple suggestions.

* `memset(g_cb_ingored_pids, 0, sizeof(g_cb_ignored_pids));`
* Remove the printk statement